protocol/src/main/kotlin/models/MessageData.kt

```diff
@@ -0,0 +1,49 @@
+package models
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import dateTimeFormat
+import typeid.TypeID
+import java.time.OffsetDateTime
+
+sealed interface MessageData
+
+class RfqData(
+  val offeringID: TypeID,
+  val payinSubunits: Int,
+  val payinMethod: SelectedPaymentMethod,
+  val payoutMethod: SelectedPaymentMethod,
+  val claims: List<String>
+) : MessageData
+
+class SelectedPaymentMethod(
+  val kind: String,
+  val paymentDetails: Map<String, Any>
```

Comment:
is `paymentDetails` in `SelectedPaymentMethod` meant to be required? The [tbdex protocol](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#selectedpaymentmethod) has it as not required but [tbdex-js](https://github.com/TBD54566975/tbdex-js/blob/9cf74c41eaff824e1746dcd40e9c544d1f8d632f/packages/protocol/src/types.ts#L127) seems to have it as required?

Reply:
Hmmmm I think it is not required...for example one method in the future might be a previously topped up stored balance, in which case you don't have any additional payment details to submit